src/main/webapp/dashboard.html

```diff
@@ -30,8 +30,22 @@
   <div class="tab-content">
     <!-- About Me -->
     <div id="about-me" class="container tab-pane active"><br>
-      <h3>About Me</h3>
-      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+      <div class="card col-12 ">
+        <div class="card-body">
+          <img class="card-img-top img-thumbnail w-25 p-2 m-2 mx-auto d-block" src="images/dior.jpg" alt="Card image">
+          <h4 id="user-name" class="card-title dark-emph">Dior<br>
+            <small id="user-location">Sunnyvale CA.</small>
+          </h4>
+          <p class="card-text">The cutest pug in The tech industry. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```

Comment:
nit: maybe we can break this line up a bit
```suggestion
          <p class="card-text">The cutest pug in The tech industry. Lorem ipsum dolor sit amet,
           consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```